src/main/java/io/mo/db/ConnectionManager.java

```diff
@@ -42,7 +42,7 @@ public static Connection getConnection(){
                 }
```

Comment:
retry 3 or 30 times at most?

Reply:
retry 3 times，and wait 10 senconds per times